_data/sites.json

```diff
@@ -4878,6 +4878,14 @@
         ]
     },
 
+    {
+        "name": "Pocket Casts",
+        "url": "https://support.pocketcasts.com/",
+        "difficulty": "hard",
+        "notes": "If the web player was purchased, access to it will be lost as well",
```

Comment:
This should state that account deletion requires an email in addition to the information already here.